_posts/2018-01-23-updates.markdown

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: Happy New Year, From Team ilios!
+date: 2018-01-23  15:00:00
+categories: updates
+---
+
+We of the Ilios team welcome you to the new year, and the first release of 2018, __v3.42.0__, which adds further permissions improvements, additional data to downloaded reports, and a calendar view for learner groups -- aloing with a bunch of other improvements to start off the year right.
```

Comment:
Oh, come on, seriously? "We of the Ilios team"? Are we the Stonecutters or something?

Regardless of how you feel about that: `along` is misspelled on this line.

Also, I would strongly recommend avoiding the ` -- `. It seems to be used incorrectly here.  And the sentence is a run-on sentence anyway, so removing the ` -- ` might help that too.

My counterproposal:

> Welcome to the New Year and the first release of 2018, __v3.42.0__. This release adds permissions improvements, enhancements to downloaded reports, a calendar view for learner groups, and a bunch of other improvements!